packages/prosemirror-lwdita-backend/tsconfig.json

```diff
@@ -5,7 +5,9 @@
     "rootDir": "src",
     "outDir": "dist",
     "module": "Node16",
-    "moduleResolution": "Node16"
+    "moduleResolution": "Node16",
+    "resolveJsonModule": true,
+    "esModuleInterop": true
```

Comment:
Why are we setting `esModuleInterop` to true??
`resolveJsonModule` Should be pushed to the top level config.

Reply:
@marmoure I have removed it from this config, but FYI, we're using this option already by importing it from the default config `@tsconfig/recommended/tsconfig.json`.
Fixed with 0cf999e